src/tls.c

```diff
@@ -11959,7 +11959,9 @@ int TLSX_PopulateExtensions(WOLFSSL* ssl, byte isServer)
             ret = TLSX_PopulateSupportedGroups(ssl, &ssl->extensions);
             if (ret != WOLFSSL_SUCCESS)
                 return ret;
+        #ifdef NO_PSK 
```

Comment:
Curious what this is addressing. Could be worth adding a comment explaining.

Reply:
That value just ends up never being read in the `!NO_PSK` case. I'll add a comment.

Reply:
Done